src/views/Hearing/index.js

```diff
@@ -188,6 +190,8 @@ class Hearing extends React.Component {
               onPostVote={onPostVote}
             />
           </div>
+          <hr/>
+          <a href={reportUrl}><FormattedMessage id="downloadReport"/></a>
```

Comment:
Yes, the data is open and available on the page and in the api, so there is no reason the report should not be downloadable too.